backend/WSS/payment.py

```diff
@@ -0,0 +1,13 @@
+from zeep import Client
+from zeep.proxy import ServiceProxy
+from django.conf import settings
+
+wsdl = settings.PAYMENT_SETTING['wsdl']
+MERCHANT = settings.PAYMENT_SETTING['MERCHANT']
+client = Client(wsdl)
```

Comment:
I highly recommend defining the Client inside the below methods since it calls some HTTP requests that are too soon to be done on file load. Instead, it's good to set up the client on demand.

Reply:
There are no requests to be done when the `Client` object is instantiated. it's just instantiating, and the `client` object can be shared to all payment and verification requests.